docs/syntax/images.md

```diff
@@ -1,29 +1,34 @@
 # Images
 
-Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with two differences:
+Images include screenshots, inline images, icons, and more. Syntax for images is like the syntax for links, with the following differences:
+
 1. instead of link text, you provide an image description
 2. an image description starts with `![` not just `[`
+3. there are restrictions on the scope of image paths
 
-Images can be referenced from the top-level `_static` dir or a local image dir.
+:::{note}
+If you reference an image using a relative path, it must be relative to the `toc.yml` or `docset.yml` rather than relative to the page that uses it.
```

Comment:
```suggestion
If a page uses a relative image path, it must be relative to the `toc.yml` or `docset.yml` that includes the page rather than relative to the page itself.
```